disperser/dataapi/prometheus_client.go

```diff
@@ -10,11 +10,15 @@ import (
 )
 
 // maxNumOfDataPoints is the maximum number of data points that can be queried from Prometheus based on latency that this API can provide
-const maxNumOfDataPoints = 3500
+const (
+	maxNumOfDataPoints = 3500
+	minWindowSizeInSec = 60
```

Comment:
IUUC this is just a rate window, not anything about "min", if so just call it ThroughputRateWindowInSec